bindings/python/htcondor2/_startd.py

```diff
@@ -1 +1,92 @@
+from typing import (
+    Union,
+    List,
+)
+
+from ._common_imports import (
+    classad,
+    Collector,
+    DaemonType,
+)
+
+from ._drain_type import DrainType
+from ._completion_type import CompletionType
+
+from .htcondor2_impl import (
+    _startd_drain_jobs,
+    _startd_cancel_drain_jobs,
+    _history_query,
+)
+
+
+class Startd():
+
+    def __init__(self, location : classad.ClassAd = None):
+        if location is None:
+            c = Collector()
+            location = c.locate(DaemonType.Startd)
+
+        if not isinstance(location, classad.ClassAd):
+            raise TypeError("location must be a ClassAd")
+
+        self._addr = location['MyAddress']
+        # We never actually use this for anything.
+        # self._version = location['CondorVersion']
+
+
+    # In version 1, check_expr and start_expr could also be `ExprTree`s.
+    def drainJobs(self,
+      drain_type : DrainType = DrainType.Graceful,
+      on_completion : CompletionType = CompletionType.Nothing,
+      check_expr : str = "True",
+      start_expr : str = "False",
```

Comment:
these should be nullptr rather than defaulted to expression strings.

Reply:
Did you mean `None`?

Reply:
(or that the C code should send up empty strings if `check_expr` or `start_expr` were not specified?)

Reply:
Punted to HTCONDOR-2163.